File: readability/readability3.c
#include <cs50.h>
#include <stdio.h>
#include <ctype.h>
#include <string.h>
#include <stdlib.h>

int count_letters(string s);

int main(void)
{
    string s = get_string("Text: ");
    int l = count_letters(s);
    printf("%i", l);
    printf(" letters\n");
}

int count_letters(string s)
{
    int i = 0;
    for (int j = 0; j < strlen(s); j++)
    {
        if isalpha(s[i])
        {
            i++;
        }
    }
    return i;
}

    //for (int j = 0, n = strlen(s); j < n; j++)
    //{
    // if((s[i] >= 65 && s[i] <= 90) || (s[i] >= 97 && s[i] <= 122))

File: sentimental-readability/readability3.py
from cs50 import get_string

text = get_string("Text: ")

i = 0
letter = 0
for i in range(len(text)):
    if (text[i].isalpha()):
        letter+=1

words = text.count(" ") + 1

sentences = text.count(".") + text.count("!") + text.count("?")

L = round(letter/words * 100)
S = (sentences/words * 100)

grade = round(0.0588 * L - 0.296 * S - 15.8)

if grade > 16:
    print("Grade 16+")
elif grade < 1:
    print("Before Grade 1")
else:
    print(f"Grade {grade}")